app/web/src/index.css

```diff
@@ -13,6 +13,13 @@
  * END --- TAILWIND GENERATOR EDIT
  */
 
+@layer base {
+  body {
+    font-family: 'Fira Sans', ui-sans-serif, system-ui, -apple-system, system-ui, "Segoe UI", "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
```

Comment:
I basically prepended Fira Sans to the default font stack, but unsure if that is unwise for some reason like unnecessary fonts being loaded. I don't think that happens with CSS though.

Reply:
This is good.